js/viz/docs/docSankey.js

```diff
@@ -0,0 +1,484 @@
+            * @name dxSankeyOptions.label.font.color
+            * @type string
+            * @default '#ffffff'
+            */
+            color: '#ffffff',
+            /**
+            * @name dxSankeyOptions.label.font.family
+            * @type string
+            * @default undefined
+            */
+            family: undefined,
+            /**
+            * @name dxSankeyOptions.label.font.weight
+            * @type number
+            * @default undefined
+            */
+            weight: undefined,
+            /**
+            * @name dxSankeyOptions.label.font.size
+            * @type number|string
+            * @default undefined
+            */
+            size: undefined,
+            /**
+            * @name dxSankeyOptions.label.font.opacity
+            * @type number
+            * @default undefined
+            */
+            opacity: undefined
+        },
+        /**
+        * @name dxSankeyOptions.label.shadow
+        * @type object
+        */
+        shadow: {
+            /**
+            * @name dxSankeyOptions.label.shadow.color
+            * @type string
+            * @default '#000000'
+            */
+            color: '#000000',
+            /**
+            * @name dxSankeyOptions.label.shadow.blur
+            * @type number
+            * @default 1
+            */
+            blur: 1,
+            /**
+            * @name dxSankeyOptions.label.shadow.opacity
+            * @type number
+            * @default 0.8
+            */
+            opacity: 0.8,
+            /**
+            * @name dxSankeyOptions.label.shadow.offsetX
+            * @type number
+            * @default 0
+            */
+            offsetX: 0,
+            /**
+            * @name dxSankeyOptions.label.shadow.offsetY
+            * @type number
+            * @default 1
+            */
+            offsetY: 1,
+        },
+        /**
+        * @name dxSankeyOptions.label.customizeText
+        * @type function(itemInfo)
+        * @type_function_param1 itemInfo: dxSankeyNodeItem
+        * @type_function_return string
+        * @notUsedInTheme
+        */
+        customizeText: undefined
+    },
+    /**
+    * @name dxSankeyOptions.tooltip
+    * @type object
+    * @inheritdoc
+    */
+    tooltip: {
+        /**
+        * @name dxSankeyOptions.tooltip.enabled
+        * @type boolean
+        * @default true
+        */
+        enabled: true,
+        /**
+        * @name  dxSankeyOptions.tooltip.customizeTooltip
+        * @default undefined
+        * @type function(info)
+        * @type_function_param1 info:object
+        * @type_function_param1_field1 type:string
+        * @type_function_param1_field2 from:string
+        * @type_function_param1_field3 to:string
+        * @type_function_param1_field4 weight:Number
+        * @type_function_param1_field5 title:string
+        * @type_function_param1_field6 weightIn:Number
+        * @type_function_param1_field7 weightOut:Number
```

Comment:
Usually, this function accepts an object describing the element to which the tooltip belongs, like a series point in the Chart or a funnel item in the Funnel. Which element do these attributes describe?